src/agents/sophia/sub-agents/watcher/agent.ts

```diff
@@ -0,0 +1,41 @@
+import { LlmAgent } from "@iqai/adk";
+import { env, model } from "../../../../env";
+import { getIqWikiTools } from "./tools";
+
+export async function watcherAgent(): Promise<LlmAgent> {
```

Comment:
I think this can be better named, its not watching per say, just checking iq wiki